solver/Cargo.toml

```
@@ -13,9 +13,10 @@ path = "src/main.rs"
 
 [dependencies]
 anyhow = "1.0"
-async-trait = "0.1"
 contracts = { path = "../contracts" }
+ethcontract = "0.9"
```

Remove default features. We already get the http transport from web3 directly. This is why there are so many changes in Cargo.lock, because there are a lot of new transitive dependencies added.
```suggestion
ethcontract = { version = "0.9", default-feature = false }
```